.circleci/config.yml

```diff
@@ -345,3 +345,31 @@ workflows:
                 - << pipeline.parameters.dev_git_branch >>
                 - << pipeline.parameters.staging_git_branch >>
                 - << pipeline.parameters.prod_git_branch >>
+
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 5 * * *"
+          filters:
+            branches:
+              only:
+                - << pipeline.parameters.dev_git_branch >>
+                - << pipeline.parameters.staging_git_branch >>
+                - << pipeline.parameters.prod_git_branch >>
+    jobs:
+      - build
+      - lint_audit:
+          requires:
+            - build
+      - test_backend:
+          requires:
+            - build
+      - cucumber_test:
+          requires:
+            - build
+      - test_frontend:
+          requires:
+            - build
```

Comment:
If the purpose of this scheduled run is to perform security scans we could remove these steps.

Reply:
I went back and forth on this. Scans are certainly the main purpose, but does it matter if we run the other tests also?

Reply:
I think the only draw back is that it may take longer to run the pipeline and the best cleanest code is no code at all. :) It's technically a waste of compute, but we're not paying for compute so it doesn't really matter. If our FE, BE or UAT tests fail here I would see that as evidence that the test itself is written incorrectly (something like an inconsistent failure from badly generated mock data).